src/global_logic.js

```diff
@@ -2500,13 +2504,28 @@ module.exports.treatSupportAbility = function (totals, chara) {
                     }
                     continue;
                 case "normalSupportKonshin":
+                    {
+                        let supportKonshinValue = module.exports.calcHaisuiValue("normalSupportKonshin", support.value, 1, totals[key]["remainHP"]);
+                        if (totals[key].isConsideredInAverage) {
+                            for (var key2 in totals) {
+                                totals[key2]["normalSupportKonshin"] = Math.max(totals[key2]["normalSupportKonshin"], supportKonshinValue);
+                            }
+                        } else {
+                            totals[key]["normalSupportKonshin"] = Math.max(totals[key]["normalSupportKonshin"], supportKonshinValue);
+                        }
+                    }
+                    continue;
+                case "normalSupportKonshin_hpDebuff":
+                    {
                     let supportKonshinValue = module.exports.calcHaisuiValue("normalSupportKonshin", support.value, 1, totals[key]["remainHP"]);
```

Comment:
this line need indent +4

Reply:
Fixed

Reply:
!? no I thought it was only single line needed indent spaces +4
now "continue" has different indent level ?

if use block scope for "case"
there is a style swith and case in the same indent level.
but I did not mention that.

Reply:
It was because cases were in it is static scope. It has been removed in my latest commit